### DISCUSSION TITLE
Rich is automatically printing API requests.

### DISCUSSION THREAD

**Discussion by P1etrodev** [Textualize/rich#3099] [Q&A] [answered]
I'm making an app using Supabase and Rich.
Everytime I make a request to the Supabase API, Rich will print some kind of log, and I don't know if this is a feature that I can disable or a bug.
I've tried uninstalling Rich and Supabase does not give any output by default.

![image](https://github.com/Textualize/rich/assets/88037480/c4adf377-9da0-43b5-915b-b078798999d4)

I've reduced the algorithm to a basic testing file. This is the only thing I do, so it's nothing I could've configured.

![image](https://github.com/Textualize/rich/assets/88037480/16e065bf-d11a-46e5-a7b8-c93bd8b3ad24)

**Comment by willmcgugan** [comment]
That output looks like it is from the logging module. Highly unlikely it is anything to do with Rich.

**Reply by P1etrodev** [reply, marked as answer]
Yes, I figured that was the problem, but I'm still not sure why it happens.
Solution was posted at this [StackOverflow's post](https://stackoverflow.com/questions/76964238/is-this-a-python-richs-feature-or-a-bug).
